src/main.rs

```diff
@@ -57,34 +52,33 @@ struct Admin;
 
 #[group]
 #[description = "Miscelenaous commands"]
-#[commands(ping)]
+#[commands(ping, link, board, paste)]
```

Comment:
Can you make a new category?
Make one called "collaboration", and  put board inside that

and one called "utility" and put  link & paste in there